src/Feed/Feed.js

```diff
@@ -19,11 +19,12 @@ export default function Feed(props) {
 
     let { content, createdAt, id, latitude, longitude, ringMinMax } = data.posts[postIndex]
     let [min, max] = ringMinMax.slice(1, -1).split(', ').map( char => +char )
+    let center = {lat: () => latitude, lng: () => longitude}
```

Comment:
Will we need to update this for just the ring number? I think drew was working on just returning the ring number